pkg/utils/progress.go

```diff
@@ -64,7 +85,7 @@ func (pp *ProgressPrinter) goPrintProgress(
 	}
 	if testWriter != nil {
 		bar.SetWriter(testWriter)
-		bar.SetRefreshRate(10 * time.Millisecond)
+		bar.SetRefreshRate(2 * time.Second)
```

Comment:
why changing this to 2s

Reply:
For testing, if keeping it 10ms, "progress = 0%" would be filled in the testing writer, hence we cannot see any progress.